### ISSUE TITLE
MKD camera_test20190222080904: Error: Not implemented

### ISSUE THREAD

**Issue by ratsputin** [svrooij/ipcam2mqtt#14]
My Reolink cameras insist on creating directories for snapshots.  When I do a test from the camera (even when I tell the camera not to use a directory), I get the following from ipcam2mqtt:

```
ERROR: <::ffff:192.168.1.201> CWD //camera_test20190222080904: Error: ENOENT: no such file or directory, stat '/srv/homeassistant/camera_test20190222080904'
ERROR: <::ffff:192.168.1.201> CWD camera_test20190222080904: Error: ENOENT: no such file or directory, stat '/srv/homeassistant/camera_test20190222080904'
ERROR: <::ffff:192.168.1.201> MKD camera_test20190222080904: Error: Not implemented
```

Is there an easy fix for this on the ipcam2mqtt side of things?  I don't even care to save the images, I just want to push them via MQTT to Home Assistant.

**Comment by svrooij** [comment]
It’s just a slimmed down ftp server, so this might be easy to implement.
Will check it out next weekend

**Comment by ratsputin** [comment]
Stephan-

Thank you for the response.  After digging into this a bit, I found that changing the line

```mkdir: noop(), ```
to
```mkdir: noopOk(),```

in server.js resolved the issue.  I did find out later, though, that all of this was unnecessary as the cameras only perform directory operations when doing a test.  Normal uploads just drop the images in the default directory.  Go figure.

Kind regards and thanks again for the response,

Brett
